src/content/en/2020/mobile-web.md

```diff
@@ -22,3 +22,278 @@ queries: 12_Mobile_Web
 #featured_stat_label_3: TODO
 unedited: true
 ---
+
+## Introduction
+We keep hearing that the Mobile Web has grown explosively in the last decade and is now the primary way many people experience the web. In this chapter we explore just how significant the mobile web is, and what kind of experiences visitors are expecting and being served.
+
+2020 has seen a big surge in [internet usage](https://www.nytimes.com/interactive/2020/04/07/technology/coronavirus-internet-use.html), on both mobile and desktop, due to the global pandemic. There has been an uptick in visits to news sites, ecommerce and social media sites -- as people across the globe adjusted to a new lifestyle with stay-at-home orders and social distancing. 2020 has been a significant year in history, for the web and for mobile usage.
+
+### <a name="datasources"></a> A note on our data sources
```

Comment:
@bazzadp what's the correct syntax for force-setting the heading IDs?

Reply:
```suggestion
### A note on our data sources {datasources}
```

However why not just use that as the header?

Reply:
and then linking just works: [link me](#datasources) ?

not sure what you mean by "why not just use that as the header"?

Reply:
If you leave it like this:

```
# A note on our data sources
```

It will automatically create an id and basically create this HTML:

```html
<h1 id="a-note-on-our-data-sources">A note on our data sources</h1>
```

No need to enter a link! It does it for you.

However you can use the `{ }` syntax if you want to override the link:

```
# A note on our data sources {barry-link}
```

will be converted to:

```html
<h1 id="barry-link">A note on our data sources</h1>
```

But then why not just call the heading Barry link? There are some rare use cases where you want to override the header but mostly let it do it's thing.

Let me see if I can stage your chapter as it is now on a test site so you can see what it looks like.

Reply:
You chapter so far is now available here: https://20201209t002313-dot-webalmanac.uk.r.appspot.com/en/2020/mobile-web so you can see what it looks like so far.

And you'll see all your header have ids and are linked from the table of contents. All from whatever markdown you have in there at the moment.

Reply:
So I would just change this to this:

```suggestion
### Data sources
```

And it should sort out your Lighthouse error

Reply:
Updated